src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, activeTabId }) => {
+  const activeTabObject = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
```suggestion
  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Don't use variable type for naming